src/adapter/src/webhook.rs

```diff
@@ -87,6 +97,21 @@ impl AppendWebhookValidator {
             secret_contents.insert(column_idx, (secret, use_bytes));
         }
 
+        // Transform any calls to `now()` into a constant representing of the current time.
+        //
+        // Note: we do this outside the closure, because otherwise there are some odd catch unwind
+        // boundary errors, and this shouldn't be too computationally expensive.
+        prep_scalar_expr(
+            &mut expression,
+            ExprPrepStyle::WebhookValidation {
+                now: to_datetime(now_fn()),
+            },
+        )
+        .map_err(|err| {
+            tracing::error!(?err, "failed to evaluate current time");
```

Comment:
Everything covered except this, any way to trigger it easily? https://buildkite.com/materialize/coverage/builds/304

Reply:
Unfortunately there is no easy way